src/booleans.js

```diff
@@ -1,61 +1,127 @@
 function negate(a) {
-  // your code here
+  if(a === true){
+    return false;
+  } else if(a === false){
+    return true;
+  }
 };
 
 function both(a, b) {
-  // your code here
+ if ((a === true) && (b === true)){
+   return true;
+ } else {
+   return false;
+ }
 };
 
 function either(a, b) {
-  // your code here
+  if ((a === true) || (b === true)){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function none(a, b) {
-  // your code here
+  if ((a !== true) && (b !== true)){
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if ((a === true) && (b === false)){
+    return true;
+  } else if ((a === false) && (b === true)) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function truthiness(a) {
-  // your code here
+  if (a){
+    return true;
+  } else{
+    return false;
+  }
 };
 
 function isEqual(a, b) {
-  // your code here
+  if (a === b){
+    return true;
+  } else {
+    return false;
+  };
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  if (a > b){
+    return true;
```

Comment:
Same here, you could remove the if statement as the condition resolves to a boolean.